test/specs/modules/Popup/Popup-test.js

```diff
@@ -107,23 +107,20 @@ describe('Popup', () => {
   })
 
   describe('positioning', () => {
-    it('is always within the viewport', () => {
-      _.each(Popup._meta.props.positions, position => {
+    POSITIONS.forEach(position => {
+      it('is always within the viewport', () => {
```

Comment:
This test work now and #1239 will work.

There is simple explanation why it worked before: test used `_meta.props.positions` while they were defined in `_meta.props.positioning`, so it doesn't really work. But, when I add real positions, it failed. `beforeEach` handler applied to `wrapperMount` once, now it applies on each `position` and everything works fine.